internal/api/apiinternal/bootstrap_test.go

```diff
@@ -1,4 +1,4 @@
-package internal
+package apiinternal
```

Comment:
This package rename happened due to the way that `internal` scoping works in go modules. Found out something new in the process of this -- even if the submodules share the same `go.mod` file and ostensibly exist in the same project, only subpackages that are either in the direct package hierarchy or siblings to an `internal` package have access to it.

Without the rename, since `commands/controller` isn't directly in the package hierarchy of this subpackage, you get build errors due to the internal naming, despite the fact that they're in the same project.